spec/models/ems_refresh/refreshers/openshift_refresher_spec.rb

```diff
@@ -117,13 +131,8 @@ def assert_specific_container_route
       :host_name => "www.example.com"
     )
 
-    @container_route.container_service.should have_attributes(
-      :name => "frontend"
-    )
-
-    @container_route.container_project.should have_attributes(
-      :ems_ref => "9f2f3e05-286d-11e5-b5bb-727174f8ab71",
-      :name    => "test"
-    )
+    @container_route.container_project.should == ContainerProject.find_by_name("test")
+    @container_route.container_service.should == ContainerService.find_by_name("frontend")
```

Comment:
you could have 2 services named "frontend", each in a separate namespace.
so this check is not very accurate.